samples/app/src/androidTest/java/com/sourcepoint/app/v6/TestData.kt

```diff
@@ -70,6 +70,7 @@ object TestData {
     var VENDOR_NAME = "google"
     var FEATURES = "Features"
     var PURPOSES = "Purposes"
+    const val CONSENT_WEB_VIEW_TAG_NAME = "consent-web-view"
```

Comment:
is there any other `const` in this file or namespace? maybe group them up somewhere?

Reply:
this speciffic const is used in tests only, and I'm not a fan of grouping up all the constants for example in one file, since we need flow separation